test/OpenFeature.Contrib.Providers.Flagd.Test/FlagdConfigTest.cs

```diff
@@ -80,6 +81,80 @@ public void TestFlagdConfigSetCertificatePath()
             Assert.False(config.UseCertificate);
         }
 
+        [Fact]
+        public void TestFlagdConfigFromUriHttp()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("http://domain:8123"));
+
+            Assert.False(config.CacheEnabled);
+            Assert.Equal(new System.Uri("http://domain:8123"), config.GetUri());
+        }
+
+        [Fact]
+        public void TestFlagdConfigFromUriHttps()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("https://domain:8123"));
+
+            Assert.False(config.CacheEnabled);
+            Assert.Equal(new System.Uri("https://domain:8123"), config.GetUri());
+        }
+
+        [Fact]
+        public void TestFlagdConfigFromUriUnix()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("unix:///var/run/tmp.sock/"));
```

Comment:
nit: another minor one (and maybe I'm missing something :sweat_smile:) but the trailing `/` here makes the socket look like a dir (which is slightly odd/confusing). I know it's just test data, so it's not a big deal.

Reply:
Tidied this up :)